charts/keycloak/ci/postgres-ha-netpol-values.yaml

```diff
@@ -0,0 +1,90 @@
+replicas: 2
+
+podLabels:
+  test-label: test-label-value
+
+podAnnotations:
+  test-annotation: test-annotation-value-{{ .Release.Name }}
+  test-int-annotation: "12345"
+
+startupScripts:
+  hello.sh: |
+    #!/bin/sh
+
+    echo '********************************************************************************'
+    echo '*                                                                              *'
+    echo '*                        Hello from my startup script!                         *'
+    echo '*                                                                              *'
+    echo '********************************************************************************'
+
+lifecycleHooks: |
+  postStart:
+    exec:
+      command:
+        - /bin/sh
+        - -c
+        - echo 'Hello from lifecycle hook!'
+
+extraEnv: |
+  - name: JGROUPS_DISCOVERY_PROTOCOL
+    value: dns.DNS_PING
+  - name: JGROUPS_DISCOVERY_PROPERTIES
+    value: 'dns_query={{ include "keycloak.serviceDnsName" . }}'
+  - name: CACHE_OWNERS_COUNT
+    value: "2"
+  - name: CACHE_OWNERS_AUTH_SESSIONS_COUNT
+    value: "2"
+  - name: KEYCLOAK_USER_FILE
+    value: /secrets/admin-creds/user
+  - name: KEYCLOAK_PASSWORD_FILE
+    value: /secrets/admin-creds/password
+  - name: KEYCLOAK_STATISTICS
+    value: all
+  - name: JAVA_OPTS
+    value: >-
+      -XX:+UseContainerSupport
+      -XX:MaxRAMPercentage=50.0
+      -Djava.net.preferIPv4Stack=true
+      -Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS
+      -Djava.awt.headless=true
+
+secrets:
+  admin-creds:
+    stringData:
+      user: admin
+      password: secret
+
+networkPolicy:
```

Comment:
This was actuall incorrect, but now is, so the test did not have any effect.

Reply:
Well spotted, I'll check that on future run